package.json

```diff
@@ -33,6 +33,7 @@
   },
   "devDependencies": {
     "@types/chrome": "0.0.103",
+    "@types/firefox-webext-browser": "^82.0.0",
```

Comment:
You won't need this with the `webextension-polyfill` package available (after you pull)